keps/sig-storage/20200120-skip-permission-change/README.md

```diff
@@ -111,6 +95,67 @@ type PodSecurityContext struct {
 
 - One of the risks is if user volume's permission was previously changed using old algorithm(which changes permission of top level directory first) and user opts in for `OnRootMismatch` `FSGroupChangePolicy` then we can't distinguish if the volume was previously only partially recursively chown'd.
 
+## Production Readiness Review Questionnaire
+
+### Feature enablement and rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [ ] Feature gate
+    - Feature gate name: ConfigurableFSGroupPolicy
+    - Components depending on the feature gate: kubelet, kube-apiserver, kube-controller-manager
+
+* **Does enabling the feature change any default behavior?**
+  No enabling the feature gate does not change any default behaviour.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**
+  Yes the feature-gate can be disabled once enabled. This will cause all volume mounts that
+  require permission change to use recursive permission and ownership change. This is the default
+  behaviour without the feature gate.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  For pods that have expected value in `pod.Spec.SecurityContext.FSGroupChangePolicy` as defined in this KEP,
+  it will start using specified policy.
+
+* **Are there any tests for feature enablement/disablement?**
+  There aren't any e2e but there are unit tests that cover this behaviour.
+
+### Rollout, Upgrade and Rollback Planning
+
```

Comment:
Can we keep the questions from the template here so that we can fill it out later?

Reply:
brought them back.

Reply:
Actually these, and the ones below, are supposed to be done for beta.

Reply:
What happens if a PRR question does not necessarily apply to a feature. For example:

> How can an operator determine if the feature is in use by workloads?**
For a feature like this one, I don't see why would an operator care if it is in user or not. And if an operator indeed does care then it could just query new field in pod. I don't see why this has to be an metric.

Reply:
I have updated these fields anyways.